mentat/transcripts.py

```diff
@@ -3,13 +3,13 @@
 import re
 from typing import TypedDict
 
-from openai.types.chat import ChatCompletionMessageParam
+from openai.types.chat import ChatCompletionContentPartParam, ChatCompletionMessageParam
 
 from mentat.logging_config import logs_path
 
 
 class UserMessage(TypedDict):
-    message: str
+    message: list[ChatCompletionContentPartParam]
```

Comment:
This actually does change how the prior messages will appear in the text transcript. e.g. I see prior messages like: `{"role": "user", "content": [{"type": "text", "text": "Describe the doodle"}]}` whereas previously the content would have been a string. This isn't breaking with previous transcipts because the jinja template handles both. I'm actually not 100% sure openai treats both the same. While you're curling could you double check that that message and `{"role": "user", "content": "Describe the doodle"}` return the same token count?